CMakeLists.txt

```
@@ -72,6 +74,30 @@ if (NOT CMAKE_ARCHIVE_OUTPUT_DIRECTORY)
   set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/lib")
 endif ()
 
+
+set(_vtk_module_wrap_python_additional_args)
+set(LIBRARY_DESTINATION "${CMAKE_INSTALL_LIBDIR}")
```

```suggestion
set(_vtk_module_library_destination "${CMAKE_INSTALL_LIBDIR}")
```

Let's use a lower case variable and let's scope its name. By convention lower case variable are used only in the current scope ... and scoping the variable name ensure it will not have any side effect.